workflows/funcscan.nf

```diff
@@ -163,11 +165,11 @@ workflow FUNCSCAN {
         AMPs
     */
     if ( params.run_amp_screening ) {
-        if ( !params.amp_skip_hmmsearch || !params.amp_skip_amplify || !params.amp_skip_ampir ) {
+        //if ( !params.amp_skip_hmmsearch || !params.amp_skip_amplify || !params.amp_skip_ampir ) {
```

Comment:
Why is this commented out? Do you always need it for AMPCombi? If so I would delete the condition entirely (rather than just commetning)

Reply:
Yes we always need it, and you are right, we should delete it